### ISSUE TITLE
Loading States - Provide clear accessibility guidance

### ISSUE THREAD

**Issue by julauxen** [grommet/hpe-design-system#3747]

**Comment by julauxen** [comment]
Added the accessibility requirements to the figma file. I will align with devs if things on terms of development should be added there as well.
